src/tokens.js

```diff
@@ -1927,6 +2003,10 @@ module.exports = [
                         name: 'tpFontBody1Size',
                         value: '16px',
                     },
+                    ios: {
+                        name: 'body1Size',
+                        value: 'CGFloat = 16',
+                    },
```

Comment:
We have an open task right now to change `body` tokens to `text` (https://github.com/thumbtack/thumbprint-tokens/issues/4#issuecomment-568092994). Should we just go ahead and use `text` for these? Will make the future breaking change easier.

Reply:
Done